Source/WebCore/style/StyleScope.cpp

```diff
@@ -89,6 +89,16 @@ Scope::~Scope()
     ASSERT(!hasPendingSheets());
 }
 
+void Scope::incrementPtrCount() const
+{
+    m_document->incrementPtrCount();
```

Comment:
Why are we forwarding it to document?
Style::Scope can be owned by either ShadowRoot or Document.
It doesn't seem correct that we're always forwarding this to Document.

Reply:
Oh, if it can be owned by both then I definitely messed this up.